content/en/xtc/monitoring/420-monitoring-configuration.md

```diff
@@ -8,36 +8,130 @@ description: >
+
+### Defining a Quality Sensor
+
+To add a new quality sensor, go to the _Quality Sensors_ tab in _Configuration_ and click _New_ at the top right. You can define a name, description, and a number of failed executions for a number of consecutive executions (for example, 1/1 raises an alert on every failure, 3/3 raises an alert if the the scenario fails 3 times in a row, 3/10 raises an alert if 3 out of 10 consecutive executions failed).
+
+{{< image src="xtc/monitoring_qualitySensors_add.png" >}}
+Adding a new quality sensor to the project.
+{{< /image >}}
+
+{{< image src="xtc/monitoring_qualitySensors_overview.png" >}}
+Quality sensors overview.
+{{< /image >}}
+
+The quality sensor will then appear in the overview table. By clicking its context menu you may **edit** the basic quality sensor data defined before, **duplicate** it in order to create another sensor with similar settings, or **delete** it (you will be asked for confirmation). To view and edit the quality sensor details (e.g. the success metrics or notification list to be alerted on failure), either click the quality sensor's name or select **view** from the context menu. 
+
+{{< image src="xtc/monitoring_qualitySensors_details.png" >}}
+Quality sensor detail view.
+{{< /image >}}
+
+In the quality sensor's details' _Configuration_ tab, you can add **metrics**, which are basically a set of conditions that must all be met or the scenario will fail. Select criteria from these groups:
+
+* **General**: Check for execution errors or scenario failures.
+* **Event**: Check the timing of [page load events]({{< relref "../../xlt/load-testing/manual/600-client-performance/#metrics-for-perceived-performance" >}}) (_DomContentLoaded_, _First Contentful Paint_, _First Paint_, _Load Event_).
+* **Web Vitals**: Check [user experience criteria](https://web.dev/articles/vitals) (_Cumulative Layout Shift_, _First Contentful Paint_, _First Input Delay_, _Interaction to Next Paint_, _Largest Contentful Paint_, _Time to First Byte_).
+* **XLT**: Check for basic runtime limits (for transactions, actions or requests) and request errors.
+
+Depending on the selected metric, the threshold you will be asked to select will be just "on occurrence", a time in milliseconds or some more specific parameter. All defined metrics can be edited or deleted afterwards using their context menu. 
+
+Not meeting one of the defined thresholds may cause an alert to be raised (remember that this happens only after the number of consecutive failures you set before), which is then sent to the set of [**notification lists**]({{< relref "#notification-lists" >}}) defined below.  
```

Comment:
A description how notification lists are assigned to a quality sensor is missing.